lms/resources/_lti_launch.py

```diff
@@ -97,7 +96,24 @@ def h_groupid(self):
         hash_object = hashlib.sha1()
         hash_object.update(self._get_param("tool_consumer_instance_guid").encode())
         hash_object.update(self._get_param("context_id").encode())
-        return f"group:{hash_object.hexdigest()}@{self._authority}"
+        return hash_object.hexdigest()
```

Comment:
I split the existing `h_groupid` property into two separate properties `h_authority_provided_id` and `h_groupid`. See commit messages

Reply:
The diff is very confusing, but the code is actually very clear